e2e_tests/footer_tests.py

```diff
@@ -0,0 +1,60 @@
+import pytest
+from playwright.sync_api import Page, expect
+
+# Define list of footer links and their expected URLS
+
+footer_links = [
+    (
+        "Terms of use",
+        "/terms-of-use",
+        "Terms of use – AYR - Access Your Records – GOV.UK ",
+    ),
+    (
+        "Privacy",
+        "/privacy",
+        "Privacy notice – AYR - Access Your Records – GOV.UK ",
+    ),
+    (
+        "Cookies policy",
+        "/cookies",
+        "Cookies – AYR - Access Your Records – GOV.UK",
+    ),
+    (
+        "Accessibility",
+        "/accessibility",
+        "Accessibility statement – AYR - Access Your Records – GOV.UK",
+    ),
+]
+
+
+@pytest.fixture
+def setup_page(page: Page):
+    page.goto("/")
+    yield page
+
+
+@pytest.mark.parametrize(
+    "link_text, expected_url, expected_title", footer_links
+)
+def test_footer_links(setup_page, link_text, expected_url, expected_title):
+    # locate and click footer link as user would
+
+    setup_page.click(f'text="{link_text}"')
+
+    # Wait for navigation and check URL is correct
+
+    setup_page.wait_for_url(expected_url, timeout=5000)
+
+    # Assertions
+
+    # Check whether the current URL matches the expected URL
+
+    expect(setup_page).to_have_url(expected_url)
```

Comment:
Could consider this assertion as redundant as we already wait for the url on line 46

Reply:
Ok will remove it @anthonyhashemi